web/client/components/map/openlayers/swipe/SliderSwipeSupport.jsx

```diff
@@ -78,11 +78,11 @@ const HSlider = ({ type, map, heightRef }) => {
 
     const onWindowResize = () => {
         setPos({x: 0, y: 0});
-        heightRef.current = map.getProperties().size[1] / 2;
+        heightRef.current = map.getProperties().size[1];
```

Comment:
I don't think we need to change this file because the `map.pixelRatio_` is applied in the EffectSupport.jsx file

Reply:
Okay let me apply the change